src/reaper_osara.cpp

```diff
@@ -3956,6 +3956,37 @@ void cmdManageTempoTimeSigMarkers(Command* command) {
 	Main_OnCommand(40256, 0); // Tempo envelope: Insert tempo/time signature change marker at edit cursor...
 }
 
+void cmdSelectItemsUnderEditCursorOnSelectedTracks(Command* command) {
+	vector<MediaItem*> items;
+	for (int i = 0; i < CountTracks(nullptr); ++i) {
+		MediaTrack* track = GetTrack(nullptr, i);
+		for (int j = 0; j < CountTrackMediaItems(track); ++j) {
+			MediaItem* item = GetTrackMediaItem(track, j);
+			items.push_back(item);
+		}
+	}
+	PreventUIRefresh(1); // SWS use this
```

Comment:
I'd prefer to document the real reason we're doing this, rather than "because SWS does it", unless we really don't know. My guess is that we do this so that we redraw the UI once for the entire operation, rather than once for every item we select.